tla/consensus/Traceccfraft.tla

```diff
@@ -130,7 +130,11 @@ DropMessages ==
     /\ l \in 1..Len(TraceLog)
     /\ UNCHANGED <<reconfigurationVars, serverVars, candidateVars, leaderVars, logVars>>
     /\ UNCHANGED <<l, ts>>
-    /\ Network!DropMessages(logline.msg.state.node_id)
+    \* Only drop messages when processing message events
```

Comment:
Because messages loss is beyond our system and comes from the system's environment (network), my motivation was to leave message loss as much as possible to non-determinism, and not restrict message loss to certain actions or node pairs.  Obviously, that would imply that the `DropMessagesTo` operator has no parameters, not even `server`. At that time, however, that resulted in massive state-space explosion beyond what TLC could handle, but, the subsequently added `ENABLED` construct could be good enough to alleviate the problem.